content/doc/book/blueocean/activity.adoc

```diff
@@ -0,0 +1,48 @@
+---
+layout: section
+title: Activity Viewer
+wip: true
+---
+:description:
+:author:
+:email: jenkinsci-docs@googlegroups.com
+:sectanchors:
+:toc:
+:imagesdir: /doc/book/resources
+:hide-uri-scheme:
+
+= The Blue Ocean Activity Viewer
+
+The Blue Ocean Activity Viewer shows the all activity related to one
+repository Pipeline.
+
+image:blueocean/activity/overview.png[Overview of the Dashboard, role=center]
+
+
+== Activity
+
+
+
+== Branches
+
+
+image:blueocean/activity/branches.png["Aborted" Status Icon, role=center]
+
+
+== Pull Requests
+
+Blue Ocean displays Pull Requests separately from branches,
```

Comment:
Might be worth a parenthetical "(or Merge Requests depending on the SCM)"

Something like that, since Pull Requests are very GitHub-centric